package.json

```diff
@@ -6,13 +6,13 @@
         "Elixir",
         "syntax"
     ],
-    "version": "0.5.1",
+    "version": "0.6.0",
```

Comment:
Can you keep the version the same? I usually only change it when I am building a new release.

Reply:
@fr1zle , I have changed the version back to 0.5.1.